spec/factories.rb

```diff
@@ -95,8 +101,8 @@ def r_str
 
   factory :relayable_entity, class: DiasporaFederation::Entities::Relayable do
     parent_guid { generate(:guid) }
-    parent_author_signature { generate(:signature) }
-    author_signature { generate(:signature) }
+    parent_author_signature { nil }
+    author_signature { nil }
```

Comment:
I think we should make the signatures `, default: nil` in the rerelayable.rb, then we can remove this two lines here.

Without default value diaspora needs to set a nil signature when it wants to send a relayable entity. And since we don't validate the signatures in the validator, we can use nil as default here.